src/main/resources/schema/ans/0.5.3/author_operation.json

```diff
@@ -0,0 +1,42 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "id": "https://raw.githubusercontent.com/washingtonpost/ans-schema/master/src/main/resources/schema/ans/0.5.3/author_operation.json",
+  "description": "An operation on a author",
+  "type": "object",
+  "allOf": [{
+    "properties": {
+      "type": {
+        "description": "Identifies this as an ANS operation",
+        "type": "string",
+        "enum": [ "author-service" ]
```

Comment:
I'm already using 'author-service' and 'site-service' in the arc-mongo-direct-service to publish these messages.  Content api already uses them as well.